gyucheol/5일차/제일_작은_수_제거하기.js

```diff
@@ -0,0 +1,8 @@
+function solution(arr) {
+    arr.splice(arr.indexOf(Math.min(...arr)),1);
+    if(arr.length < 1)  return[-1];
+    return arr;
+
+    
+}  
```

Comment:
삼항 연산자를 이용해서 가독성 좋게 풀 수 있습니다.

```suggestion
function solution(arr) {
    arr.splice(arr.indexOf(Math.min(...arr)),1);
   
    return arr.length ? arr : [-1];
} 
```